extensions/amp-call-tracking/amp-call-tracking.md

```diff
@@ -0,0 +1,68 @@
+<!---
```

Comment:
It'd be good to cross reference the design doc and this PR because this is such a new component.